### DISCUSSION TITLE
1.17.0

### DISCUSSION THREAD

**Discussion by xaoxuu** [xaoxuu/hexo-theme-stellar#191] [Announcements]
- `A` 支持搜索插件 [hexo-generator-search](https://github.com/wzpan/hexo-generator-search) `npm i hexo-generator-search`
- `M` 文章分页优化（建议更新到 hexo 6.3.0+ 版本）

<hr /><em>This discussion was created from the release <a href='https://github.com/xaoxuu/hexo-theme-stellar/releases/tag/1.17.0'>1.17.0</a>.</em>

**Comment by Achuan-2** [comment]
更新后反而用不了搜索了
![image](https://user-images.githubusercontent.com/60436214/204317806-2d7951a7-ee79-40b2-8522-6df0aa97d8d7.png)

**Comment by xaoxuu** [comment]
我用 `blog test` 在官方 demo 博客中测试是正常的。。。
看报错位置应该是 content 为空？要不你改成 `field: post` 看看行不行

**Reply by Achuan-2** [reply]
改成post就正常了

**Reply by xaoxuu** [reply]
那应该是你有的 page 里面没有内容，我加个判断应该就好了。
